src/api/loginApi.jsx

```diff
@@ -0,0 +1,25 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+export const loginApi = createApi({
+  reducerPath: "loginApi",
+  baseQuery: fetchBaseQuery({
+    // eslint-disable-next-line no-undef
+    baseUrl: process.env.REACT_APP_API_URL,
+  }),
+  tagTypes: ["login"],
+  endpoints: (builder) => ({
+    login: builder.mutation({
+      query: (access_token) => ({
+        url: "/login",
```

Comment:
ad constant for API paths - "\login" and all other the use it

Reply:
done